modules/service/src/main/resources/lucuma/odb/graphql/OdbSchema.graphql

```diff
@@ -4699,6 +4695,9 @@ type Query {
 
   # Selects the first `LIMIT` matching observations based on the provided `WHERE` parameter, if any.
   observations(
+
+    programId: ProgramId
```

Comment:
At some point we decided that top-level queries for observations needed to specify the program id. I think we should get rid of these anyway and make the user traverse down to the program.